src/scanpy/plotting/_anndata.py

```diff
@@ -2173,17 +2188,16 @@ def _plot_gene_groups_brackets(
     from matplotlib.path import Path
 
     # get the 'brackets' coordinates as lists of start and end positions
-
-    left = [x[0] + left_adjustment for x in group_positions]
-    right = [x[1] + right_adjustment for x in group_positions]
+    left = [x[0] + left_adjustment for x in var_groups.positions]
+    right = [x[1] + right_adjustment for x in var_groups.positions]
 
     # verts and codes are used by PathPatch to make the brackets
     verts = []
     codes = []
     if orientation == "top":
         # rotate labels if any of them is longer than 4 characters
-        if rotation is None and group_labels:
```

Comment:
I guesss there was no way for `group_labels` to be `None` in practice (at least judging by the typing)?

Reply:
asking re: why you dropped the check

Reply:
Before, there were many checks in the code like `if var_groups_?? is None or len(var_groups_??) == 0`, but the validation now takes care that neither can be the case. Either `var_groups is None` or it contains nonempty sequences

Reply:
Ah and fun fact: the `wide` parameter I added was because there was some code drift between the two versions of bracket drawing code, and our tests *did* catch that!

So actually maybe we should decide on one version and update the test images.

Reply:
Got it